about.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="stylesheets/styles.css">
+  <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css"
+  href="https://fonts.googleapis.com/css?family=Lobster|Harmattan|Josefin+Sans|Abel|Lato|Playfair+Display">
+  <title>About Me</title>
+</head>
+
+<body id="page3">
+  <header>
+    <div class="header-name">
+      <a href="about.html" target="_blank">Haby Randall</a>
```

Comment:
You might want to rethink using target="_blank" for links to pages of your website since it opens pages into a new tab. Makes sense for your links to social media sites!